nautobot/core/ui/object_detail.py

```diff
@@ -219,47 +299,73 @@ class _ObjectDetailAdvancedTab(Tab):
     tab_id = "advanced"
     tab_label = "Advanced"
 
-    def __init__(self):
-        super().__init__(
-            layouts=(
-                LayoutTwoColumn(
-                    left_panels=(
-                        # TODO
-                        ObjectFieldsPanel(
-                            label="Object Details",
-                            fields=["id", "natural_slug", "slug"],
-                            ignore_nonexistent_fields=True,
-                        ),
-                        ObjectFieldsPanel(
-                            label="Data Provenance",
-                            fields=["created", "last_updated"],
-                            ignore_nonexistent_fields=True,
-                            # TODO add created_by/last_updated_by derived fields and link to REST API
-                        ),
-                        # TODO add advanced_ui custom fields and relationships
-                    ),
+    def __init__(self, **kwargs):
+        kwargs.setdefault("weight", 1000)
```

Comment:
Doesn't advanced tab always come second?

Reply:
Probably it should, good call.